src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -62,7 +62,7 @@ const NavSearchBar = () => {
       venue?.rooms
         ?.filter(
           (room) =>
-            room.type !== RoomTypes.unclickable &&
+            room.type !== (RoomTypes.unclickable || RoomTypes.iframe) &&
```

Comment:
Thanks — for the time being I've implemented with `COVERT_ROOM_TYPES`, which I think sounds cool and captures the slightly beneath the surface aspect.

Reply:
I've implemented the logic as: `(!room.type || !COVERT_ROOM_TYPES.includes(room.type)) &&` because otherwise I encountered the error that `room.type` may be `undefined`

Reply:
Sounds good.